nx/lib/nx/shared.ex

```diff
@@ -249,6 +250,56 @@ defmodule Nx.Shared do
       false
   end
 
+  @doc """
+  Approximation for the inverse error function.
+
+  ## Examples
+
+    iex> Nx.erf_inv(0.999)
+    #Nx.Tensor<
+      f64
+      2.326753756865462
+    >
+
+    iex> Nx.erf_inv(0.01)
+    #Nx.Tensor<
+      f64
+      0.008862500728738846
+    >
```

Comment:
Please indent with four spaces!

Reply:
Also, we shouldn't be testing the `Nx` API via `Nx.Shared` doctests. We should either test the Nx.Shared API directly or add the test to `nx/shared_test`. :)

Reply:
Good point. I missed that.